apiserver/facades/client/modelmanager/modelmanager.go

```diff
@@ -220,9 +221,26 @@ func (m *ModelManagerAPI) CreateModel(ctx context.Context, args params.ModelCrea
 			return result, errors.Trace(err)
 		}
 		creationArgs.Credential = credential.KeyFromTag(cloudCredentialTag)
+	} else {
+		// we're looking up credentials for the apiUser - what if JIMM is creating a model on behalf of another user (JIMM always
```

Comment:
Do we need this comment?

Reply:
i think it's worth keeping so we don't forget that JIMM always creates model on behalf of other users and does so as `admin` user